base/show.jl

```diff
@@ -469,6 +469,24 @@ end
 isidentifier(s::Symbol) = isidentifier(string(s))
 
 isoperator(s::Symbol) = ccall(:jl_is_operator, Cint, (Cstring,), s) != 0
+
+"""
+    operator_precedence(s::Symbol)
+
+Return an integer representing the precedence of operator `s`, relative
+to other operators. Higher numbers take precedence over lower-numbered
+operators. Return `0` if `s` is not a valid operator.
```

Comment:
Perhaps "Higher-numbered operators take precedence over lower-numbered operators." for consistency? (Fell by the wayside in another comment thread.)